libcloudforensics/providers/azure/internal/common.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common utilities."""
+
+import binascii
+import logging
+import os
+import re
+
+from typing import Any, List, Dict, Optional, TYPE_CHECKING
+from azure.common.credentials import ServicePrincipalCredentials  # pylint: disable=import-error
+
+if TYPE_CHECKING:
+  # TYPE_CHECKING is always False at runtime, therefore it is safe to ignore
+  # the following cyclic import, as it it only used for type hints
+  from libcloudforensics.providers.azure.internal import compute  # pylint: disable=cyclic-import
+
+# pylint: disable=line-too-long
+# See https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules
+# pylint: enable=line-too-long
+REGEX_DISK_NAME = re.compile('^[\\w]{1,80}$')
```

Comment:
```suggestion
REGEX_DISK_NAME = re.compile('^\w{1,80}$')
```

Reply:
While identical, having a double `\\` satisfies PEP 8 escape sequence (IDE complaining otherwise). I don't mind removing it and shutting off my PEP8 detector. Wdyt?